applications/osb-portal/src/components/workspace/drawer/WorkspaceResourceBrowser.tsx

```diff
@@ -111,7 +111,7 @@ const WorkspaceResourceBrowser = (props: WorkspaceProps) => {
           .map((resource: WorkspaceResource, idx: number) => (
             <TreeItem
               icon={undefined}
-              key={idx + ''}
+              key={`${idx}-${resource.name}`}
```

Comment:
This is good, but it would be better to use the resource id (only id, without index)